tasklist/client/src/v2/Tasks/AvailableTasks/index.tsx

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
I see this component identical to the v1 version, do we plan to display it differently for v2 in the future?

Reply:
The migration is done on the following PR